From d049e7d390e2d739dfe78898fbfa765d8983536f Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Mon, 30 Oct 2023 19:25:27 -0400
Subject: [PATCH 35/44] Add a prop to use linear backlight control

---
 .../settings/brightness/BrightnessController.java      | 10 ++++++++--
 1 file changed, 8 insertions(+), 2 deletions(-)

diff --git a/packages/SystemUI/src/com/android/systemui/settings/brightness/BrightnessController.java b/packages/SystemUI/src/com/android/systemui/settings/brightness/BrightnessController.java
index ee566ebf0c2d..e43d803cc65b 100644
--- a/packages/SystemUI/src/com/android/systemui/settings/brightness/BrightnessController.java
+++ b/packages/SystemUI/src/com/android/systemui/settings/brightness/BrightnessController.java
@@ -355,9 +355,15 @@ public class BrightnessController implements ToggleSlider.Listener, MirroredBrig
                 : MetricsEvent.ACTION_BRIGHTNESS;
         minBacklight = mBrightnessMin;
         maxBacklight = mBrightnessMax;
-        final float valFloat = MathUtils.min(
+        float valFloat = MathUtils.min(
                 convertGammaToLinearFloat(value, minBacklight, maxBacklight),
                 maxBacklight);
+        if (android.os.SystemProperties.getBoolean("persist.sys.phh.linear_brightness", false)) {
+            android.util.Log.e("PHH", "Linear brightness val " + value + " from " + minBacklight + " to " + maxBacklight + " makes " + valFloat);
+            valFloat = value / 65536.0f;
+        }
+        final float finalValFloat = valFloat;
+
         if (stopTracking) {
             // TODO(brightnessfloat): change to use float value instead.
             MetricsLogger.action(mContext, metric,
@@ -374,7 +380,7 @@ public class BrightnessController implements ToggleSlider.Listener, MirroredBrig
         if (!tracking) {
             AsyncTask.execute(new Runnable() {
                     public void run() {
-                        mDisplayManager.setBrightness(mDisplayId, valFloat);
+                        mDisplayManager.setBrightness(mDisplayId, finalValFloat);
                     }
                 });
         }
-- 
2.34.1

