From d62207875014103f3aed3dd1f6ee980337bb8d09 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Mon, 29 Jan 2024 17:07:07 -0500
Subject: [PATCH 37/44] Add Samsung sysinput

---
 services/core/Android.bp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/services/core/Android.bp b/services/core/Android.bp
index af42ddde6886..72fc3f8fe81a 100644
--- a/services/core/Android.bp
+++ b/services/core/Android.bp
@@ -268,6 +268,7 @@ java_library_static {
         "vendor.aospa.biometrics.face",
 
         //AIDL
+        "vendor.samsung.hardware.sysinput-V1-java",
         "vendor.samsung.hardware.biometrics.fingerprint-V1-java",
         "vendor.samsung.hardware.light-V1-java",
     ],
-- 
2.34.1

