From 1d938a85fa50c86909e196a285f9fbf990bace00 Mon Sep 17 00:00:00 2001
From: Pierre-Hugues Husson <phh@phh.me>
Date: Wed, 11 Mar 2020 14:02:35 +0100
Subject: [PATCH 5/5] Every voldmanaged storage is adoptable

---
 main.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/main.cpp b/main.cpp
index 261f4d4..e93b9ad 100644
--- a/main.cpp
+++ b/main.cpp
@@ -275,7 +275,7 @@ static int process_config(VolumeManager* vm, VoldConfigs* configs) {
             int partnum = entry.partnum;
             int flags = 0;
 
-            if (entry.is_encryptable()) {
+            if (entry.is_encryptable() || true) { // why not just remove entry.is_encryptable() check here? the || true negates it anyways
                 flags |= android::vold::Disk::Flags::kAdoptable;
                 configs->has_adoptable = true;
             }
-- 
2.39.2

